packages/now-cli/test/integration.js

```diff
@@ -519,6 +519,27 @@ test('Deploy `api-env` fixture and test `now env` command', async t => {
   await nowEnvLsIsEmpty();
 });
 
+test('deploy with metadata containing "=" in the value', async t => {
+  const target = fixture('redirects-v2');
+
+  const { exitCode, stderr, stdout } = await execa(
+    binaryPath,
+    [target, ...defaultArgs, '--confirm', '--meta', 'someKey=='],
+    { reject: false }
+  );
+
+  t.is(exitCode, 0, formatOutput({ stderr, stdout }));
+
+  const url = stdout.split('https://'.lenght);
```

Comment:
`lenght` 😨 

But also, the split shouldn't be necessary at all (if it is, then there's a bug):

```suggestion
  const url = stdout;
```

Reply:
It might need `.trim()` if there's a trailing newline.

Reply:
@TooTallNate It's necessary because I'm doing a req to get the deployment and it needs the deployment "host".

Reply:
```suggestion
  const url = stdout.split('https://'.length);
```

Reply:
I see now that you're removing the prefixed `https://`, so disregard my suggestion. But fix the typo 😄